thunder/tests/distributed/test_fsdp.py

```diff
@@ -866,6 +866,65 @@ def is_same_across_ranks(t):
         return comparison_exceptions
 
 
+def _test_fsdp_transformer_engine_bucketing(input_data):
```

Comment:
I think it should be possible to construct a test with a mock test symbol like this test creates a `sincos` operation https://github.com/Lightning-AI/lightning-thunder/blob/86cd9c3b5824064f8faa024ae0427fb297c04add/thunder/tests/test_grad.py#L740
But I'd say we don't need to go this far and this test in combination with `test_fsdp_transformer_engine_bucketing` is good enough.